queries/nu/highlights.scm

```diff
@@ -286,6 +289,9 @@ key: (identifier) @property
 
 (long_flag_identifier) @attribute
 
+(attribute
+  (attribute_identifier) @attribute)
+
```

Comment:
Can't you just do `(attribute) @attribute` then?

Reply:
Do you mean `(attribute_identifier) @attribute` ?

Reply:
No, I meant what I wrote. Asked another way: what is left of the `(attribute)` node that you are _not_ capturing in these two patterns?

(It's always good to show an example -- code snippet, parsed tree, and before/after screenshot -- for any proposed query change.)

Reply:
@clason 
![image](https://github.com/user-attachments/assets/71a32d5a-be02-4719-ad79-1feddb328d68)

The `identifier` here is `example`, the rest is also in node `attribute`

Reply:
Then probably
```lisp
(attribute
  "@" @attribute
  (attribute_identifer) @attribute)
```
(please test first). If that doesn't work, at least keep the two patterns next to each other.

Reply:
That makes sense, one problem, there are similar issues for `long_flag` , `short_flag`,  `param_long_flag` etc. Do I need to change them in this PR?

Reply:
No; if you want to clean them up, that would be welcome, but should be a separate PR.

Reply:
Oh, unless you're talking about the `"@"` highlight not being consistently applied otherwise.

In that case, it would be better to refactor things here -- but I think then the right approach would be to do
```
(attribute
  "@" @attribute)
  
[
  (attribute_identifier)
  (long_flag_identifier)
  ...
] @attribute
```

(assuming this still does the right thing).